✨(marsha) cronjob running clean_medialive_dev_stack command

lunika opened the pull request:
## Purpose

run command clean_medialive_dev_stack added in PR #1108
https://github.com/openfun/marsha/pull/1108

## Proposal

- [x] add cronjob running clean_medialive_dev_stack command

lunika committed:
✨(marsha) cronjob running clean_medialive_dev_stack command

run command clean_medialive_dev_stack added in PR #1108
https://github.com/openfun/marsha/pull/1108

sampaccoud commented on apps/marsha/templates/services/app/cronjob_clean_medialive_dev_stack.yml.j2:
```diff
@@ -0,0 +1,75 @@
+                - name: POSTGRES_PORT
+                  value: "{{ marsha_postgresql_port }}"
+                - name: DJANGO_ALLOWED_HOSTS
+                  value: "{{ marsha_host | blue_green_hosts }}"
```
Don't set it as this is a cron job?

lunika replied:
:+1:

sampaccoud commented on apps/marsha/templates/services/app/cronjob_clean_medialive_dev_stack.yml.j2:
```diff
@@ -0,0 +1,75 @@
+                  value: "{{ marsha_postgresql_port }}"
+                - name: DJANGO_ALLOWED_HOSTS
+                  value: "{{ marsha_host | blue_green_hosts }}"
+                - name: DJANGO_CLOUDFRONT_PRIVATE_KEY_PATH
```
Don't set it as this is a cron job?

lunika replied:
:+1:

lunika committed:
🔥(marsha) remove useless env variables in command templates

All management command added in marsha are using two environment
variables unneeded and containing sensitive data. We can safely remove
them.
